pkgs/development/python-modules/grep-ast/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "grep-ast";
+  version = "3cf6200";
+
+  src = fetchFromGitHub {
+    owner = "paul-gauthier";
+    repo = "grep-ast";
+    rev = version;
```

```suggestion
    rev = "3cf620091364e95e501fa2fd1125cc9fae7971e5";
```
please use the _full_ commit hash